lint-css.sh

```diff
@@ -1,34 +1,11 @@
 #!/bin/bash
 
-while getopts 'c:p:n:t:f:' flag; do
-	case "${flag}" in
-		c) cwdiswppslinter=${OPTARG} ;;
-		p) plugindirname=${OPTARG} ;;
-		n) namespace=${OPTARG} ;;
-		t) textdomain=${OPTARG} ;;
-		f) fix=${OPTARG} ;;
-	esac
-done
-
-# Get the absolute path to the plugin we want to check.
-if [ "$cwdiswppslinter" = "1" ]; then
-	plugindir="$(dirname "$(dirname "$(dirname "$(dirname "$(realpath "$0")" )" )" )" )/$plugindirname"
-	wpcontentdir="./../../../../"
-	scriptsdir="$(dirname "$(realpath "$0")" )/"
-else
-	cwdiswppslinter=0
-	plugindir="$(dirname "$(dirname "$(realpath "$0")" )" )"
-	wpcontentdir="$(dirname "$(dirname "$(dirname "$(dirname "$(realpath $0)" )" )" )" )"
-	scriptsdir="$plugindir/wpps-scripts/"
-fi
-
-#Go to wp-content directory.
-cd "$wpcontentdir";
-sh "${scriptsdir}install-script-dependencies.sh" -c $cwdiswppslinter
+# Run setup.
+. ./setup.sh
 
 # Run the lint command from the wp-content directory.
 if [ "$fix" = "1" ]; then
-	npm run lint:css "$plugindir/**/*.*css"  -- --fix;
+	npm run lint:css "$plugindir/**/css/src/*.*css"  -- --fix;
 else
-	npm run lint:css "$plugindir/**/*.*css";
+	npm run lint:css "$plugindir/**/css/src/*.*css";
```

Comment:
This is the main thing that fixed `npm run lint:css`. It was linting a built file.

Reply:
I removed the build from fse-studio preinstall now. But this is probably good to keep anyway!

Reply:
Oh nice!